### PR TITLE
Add Node.js sample

### PR DESCRIPTION
**Pull request by ctoyan**

### CODE REVIEW

File: samples/nodejs/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const { Issuer } = require('openid-client')
+
+;(async () => {
+  const app = express()
+  const PORT = argv.port || 8000
+  const ISSUER = argv.issuer || 'https://api.mpin.io'
+  const CLIENT_ID = argv.clientId
+  const CLIENT_SECRET = argv.clientSecret
+  const REDIRECT_URL = argv.redirectUrl || 'http://localhost:8000/login'
+  console.log(argv)
```
Comment (todinov) [inline samples/nodejs/index.js]: I don't think this console.log is necessary.

File: samples/nodejs/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+;(async () => {
+  const app = express()
+  const HOST = process.env.HOST || '0.0.0.0'
+  const PORT = process.env.PORT || 8000
+  const ISSUER = process.env.ISSUER || 'https://api.mpin.io'
+  const CLIENT_ID = process.env.CLIENT_ID
+  const CLIENT_SECRET = process.env.CLIENT_SECRET
+  const REDIRECT_URL = process.env.REDIRECT_URL || 'http://localhost:8000/login'
+
+  console.log('CLIENT_ID', CLIENT_ID)
```
Comment (valdikamenarov) [inline samples/nodejs/index.js]: do we need this `console.log`?

Comment (ctoyan) [reply samples/nodejs/index.js]: Nope. Removing it.

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+name: nodejs
+
+on:
+  push:
+    paths:
+      - "samples/nodejs/*"
+      - ".github/workflows/nodejs.yml"
```
Comment (todinov) [inline .github/workflows/nodejs.yml]: Can you add this for the other sample too, so the workflow is triggered when it's changed.

File: samples/go/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,8 @@
 FROM golang:alpine
 WORKDIR /src
 COPY . .
+
 RUN go build -o sample .
-ENTRYPOINT ["./sample"]
+RUN chmod +x ./build-command.sh
```
Comment (todinov) [inline samples/go/Dockerfile]: Can we call the script entrypoint.sh, build command is a bit confusing.

**Pull request merged**